portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,39 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
Usually the order is Helvetica, Arial, sans-serif. Helvetica first on those machines that have it (i.e. Macs), Arial as a fallback (Windows, Linux), sans-serif as a fallback for that (Linux).